grazer/src/main/java/edu/asu/diging/grazer/web/PersonController.java

```diff
@@ -9,24 +9,26 @@
 import org.springframework.web.bind.annotation.PathVariable;
 import org.springframework.web.bind.annotation.RequestMapping;
 
-import edu.asu.diging.grazer.core.conceptpower.IConceptpowerConnector;
+import edu.asu.diging.grazer.core.conceptpower.IConceptpowerCache;
 import edu.asu.diging.grazer.core.graphs.IGraphManager;
 import edu.asu.diging.grazer.core.model.IConcept;
 import edu.asu.diging.grazer.core.model.impl.Graph;
 
 @Controller
 public class PersonController {
     
-    @Autowired
-    private IConceptpowerConnector connector;
+    private final String URI_PREFIX = "http://www.digitalhps.org/concepts/";
     
     @Autowired
     private IGraphManager graphManager;
     
+    @Autowired
+    private IConceptpowerCache cache;
+    
     @RequestMapping(value = "/concept/{personId}", produces = MediaType.TEXT_HTML_VALUE)
     public String showPerson(@PathVariable("personId") String personId, Model model) throws IOException {
-        
-        IConcept concept = connector.getConcept(personId);
+        String uri = URI_PREFIX + personId;
```

Comment:
This should not be hardcoded. Also, uris might change, or there might be concepts from more than one source. Instead, you should retrieve concepts by their id here.